app/models/github_notification_monitor.rb

```diff
@@ -53,12 +53,21 @@ def process_notification(notification)
   end
 
   def process_issue_thread(issue)
+    process_issue_title(issue)
     process_issue_comment(issue, issue.author, issue.created_at, issue.body)
     issue.comments.each do |comment|
       process_issue_comment(comment.issue, comment.author, comment.updated_at, comment.body)
     end
   end
 
+  def process_issue_title(issue)
+    if issue.title_indicates_wip?
+      issue.add_labels([GitHubApi::Label.new(nil, "wip", nil)]) unless issue.applied_label?("wip")
```

Comment:
I wonder if this would be cleaner to call the `add_labels` method below with something like `add_labels("wip", issue.author, issue)`